### DISCUSSION TITLE
Playtime: Activate record options for all clip rows and columns as default without the need to arm the track in the mixer in order to record clip (like in Ableton but better)

### DISCUSSION THREAD

**Discussion by joungle123** [helgoboss/helgobox#1253] [Ideas]
It would be nice if we would not need to click ‘record button in mixer’ in order to activate record option in clips, it would save a lot of time to always have record buttons visible and be able to click on the ‘clip section’ at the top, and they could automatically (when clicking to record a clip, the would automatically ‘arm to record’ the whole track meaning there would be a really huge convenience for this as we would not need to move mouse to first select to record (to arm) the track we want and the drag the mouse to select to record a clip. It would be just instantaneous and would make it super fast.
![image](https://github.com/user-attachments/assets/f8eb9f67-cd08-40de-80d5-37344f86ebbd)

**Comment by helgoboss** [comment]
Actually Playtime worked like that at first. But later I introduced the necessity to arm. Because it's standard.

Before thinking further about that:

- Wouldn't it be an alternative to build a project template where all column tracks are armed by default?
- Do you want to use this for MIDI or audio recordings? What's the general usage scenario for that? Live?
- How should it work for connected controllers? It would be very easy then to record accidentally if each empty slot triggers a recording automatically, regardless of the track arm state.

**Reply by joungle123** [reply]
@helgoboss Okay you made me think deeper about this and it totally makes sense why we would need to have it such a way as it is now because possibly due to the need to hear the sound before recording (meaning - play a sound -> then record it) makes a lot of sense and in fact there is no need to change anything here. Sorry for confusion.
